tests/test_emr.py

```diff
@@ -459,6 +459,19 @@ def test_must_be_integer_in_config(self):
             self.assertRaises(ParamValidationError, self.run_and_get_cluster)
 
 
+class VisibleToAllUsersTestCase(MockBoto3TestCase):
+
+    def test_defaults(self):
+        cluster = self.run_and_get_cluster()
+        self.assertEqual(cluster['VisibleToAllUsers'], True)
+
```

Comment:
You should also test `--extra-cluster-param VisibleToAllUsers=false`

Reply:
Added.